manim/mobject/text/text_mobject.py

```diff
@@ -428,6 +436,10 @@ def __init__(
     ) -> None:
 
         self.line_spacing = line_spacing
+        if warn_missing_font:
+            fonts_list = manimpango.list_fonts()
+            if font not in fonts_list:
+                logger.warning(f"Font {font} not in {fonts_list}.")
```

Comment:
I think it is unintuitive if a warning is logged when the user doesn't specify a font. The current default argument for `font` is the empty string `""`, should this be special-cased?

(This is also the reason why the doctests are currently failing in the pipeline.)

Reply:
Agreed. Fixed.